Only allow* as the entire left-most DNS component (#9)
richsalz/draft-ietf-uta-rfc6125bis: draft-ietf-uta-rfc6125bis.md

### Checking of Wildcard Certificates {#verify-domain-wildcards}

> A client MAY match the

Reviewer: I'm concerned that this MAY could cause an interoperability problem: if clients aren't required to observe the wildcard, it is generally not useful to publish.

Author: Clients must be free to ignore wildcards, because of things like `*.air-traffic-control.aero` and others that are on the public suffix list.

reference identifier against a presented identifier whose DNS domain name

> portion contains the wildcard character `*` in a label

Reviewer: ```suggestion
portion contains the wildcard character `*` as the left-most label
```

Author: I wanted to keep the "prose description, followed by explicit rules" construction.

(following the description of labels and domain names in {{DNS-CONCEPTS}}),
provided these requirements are met:

1. There is only one wildcard character.

2. The wildcard character appears only as the content of the
   left-most label.

> 

Reviewer: Items 1 and 2 might be redundant with my wording suggestion above.

Author: I prefer to keep the explicit list of rules, easier for implementors :)

## Wildcard Certificates {#security-wildcards}

Wildcard certificates, those that have an identifier with
`*` as the left-most DNS label,
automatically vouch for any single-label host names

> within their domain, but not multiple levels.

Reviewer: ```suggestion
within their domain, but not multiple levels of labels.
```

Author: added "of labels" thanks.
